unittests/InstructionCountCI/VEX_map1.json

```diff
@@ -1019,15 +989,14 @@
       ]
     },
     "vpacksswb xmm0, xmm1, xmm2": {
-      "ExpectedInstructionCount": 3,
+      "ExpectedInstructionCount": 2,
       "Optimal": "No",
```

Comment:
Might what to double check if SVE saturate narrow can be more optimal, otherwise this might be.